lib/maintainers.nix

```diff
@@ -54,4 +54,10 @@
     githubId = 66408983;
     name = "svl";
   };
+  b3nb5n = {
+    discord = "b3nb5n";
+    github = "b3nb5n";
+    githubId = 67534814;
+    name = "b3nb5n";
```

Comment:
Up to you, but usually `name` would be a "real" name, i.e. "Ben".

Also, you should include an email; the email you use in git commits should be fine, since that's public anyway.